resources/views/platform/components/game/achievements-list-meta.blade.php

```diff
@@ -0,0 +1,35 @@
+@props([
+    'isOfficial' => true,
+    'numAchievements' => 0,
+    'numMissableAchievements' => 2,
```

Comment:
This doesn't appear to be getting passed through. Every page I visit has 2 missable achievements

Reply:
Resolved in https://github.com/RetroAchievements/RAWeb/pull/2077/commits/2792b2efbe4231578b6970fb796742c4d9725a39.